src/main/java/cheese/Parser.java

```diff
@@ -0,0 +1,81 @@
+package cheese;
+
+import cheese.command.AddCommand;
+import cheese.command.Command;
+import cheese.command.ExitCommand;
+import cheese.command.FindCommand;
+import cheese.command.ListCommand;
+import cheese.command.MarkCommand;
+import cheese.command.UpdateCommand;
+import cheese.task.Deadline;
+import cheese.task.Event;
+import cheese.task.Task;
+import cheese.task.ToDo;
+
+/**
+ * Parses user input
+ */
+public class Parser {
+    /**
+     * Get idx of item in list from user input
+     * @param inputTokens input from user
+     * @return int
+     * @throws CheeseException Missing/Incorrect input
+     */
+    public static int getIdx(String[] inputTokens, int size) throws CheeseException {
```

Comment:
Could idx be a potentially confusing name? Perhaps consider extending it to index.